README.md

```diff
@@ -40,3 +40,8 @@ Once you start feeling more comfortable, you can find more open source projects
 * [Pull Request Roulette](http://www.pullrequestroulette.com/)
 * [Code Triage](https://www.codetriage.com/)
 * [24 P[ULL] R[EQUEST]S](https://24pullrequests.com/)
+
+Links to tutorial on youtube for learning Git:
+* https://www.youtube.com/watch?v=SWYqp7iY_Tc
+* https://www.youtube.com/watch?v=WbwIoQYP6no
+* https://www.youtube.com/watch?v=HVsySz-h9r4
```

Comment:
Please do not use bare URLs, instead use the standard markdown format, for example, [name of video, from channel-name ](link)

Reply:
Also, these video links can be placed immediately after the resources already present in the README.md.

Reply:
I hope you've ensured that these are quality resources and will be helpful for people.

Reply:
Okay I will make changes and these resources are good and also people learn much faster by looking at practical rather than theory

Reply:
Yes, you're correct. Videos are much more helpful.